index.js

```diff
@@ -63,3 +64,4 @@ module.exports = {
         },
     ],
 };
+/* eslint-disable quote-props */
```

Comment:
enable?

Reply:
Doh you're right. But I decided to just add an override instead.